test/verify/test_slice_step_reverse.cpp

```diff
@@ -11,8 +11,7 @@ struct test_slice_step_reverse : verify_program<test_slice_step_reverse>
         migraphx::program p;
         auto* mm = p.get_main_module();
         migraphx::shape s{migraphx::shape::int32_type, {7, 5}};
-        auto x = mm->add_parameter("x", s);
-        mm->add_literal({{migraphx::shape::int32_type, {2}}, {-2, 2}});
+        auto x         = mm->add_parameter("x", s);
         auto slice_out = mm->add_instruction(
             migraphx::make_op("slice", {{"axes", {0, 1}}, {"starts", {0, 2}}, {"ends", {2, -1}}}),
             x);
```

Comment:
Maybe we can add another unit test in which "reverse" is before "step".

Reply:
👍🏼  Done!